Content.Client/GlobalVerbs/ViewVariablesVerb.cs

```diff
@@ -12,17 +12,19 @@ namespace Content.Client.GlobalVerbs
     [GlobalVerb]
     class ViewVariablesVerb : GlobalVerb
     {
-        public override string GetText(IEntity user, IEntity target) => "View variables";
-        public override string GetCategory(IEntity user, IEntity target) => "Debug";
-
         public override bool RequireInteractionRange => false;
 
-        public override VerbVisibility GetVisibility(IEntity user, IEntity target)
+        public override void GetData(IEntity user, IEntity target, VerbData data)
```

Comment:
What's the motivation for making `data` an argument, rather than a return value? Performance?

Reply:
More convenient than starting off every `GetData` override with `var data = new VerbData();`

Reply:
You could `return` an object initializer for convenience. I personally think passing in an argument to write values to will be confusing from an API design perspective - maybe it could be better highlighted in the docs?

Reply:
My idea was that this makes it easier to incrementally update the verb data, for example when the icon is always the same but the text/enabled state can change based on context. With object initializers you'd need to specify that twice.

I'll try to improve the docs though.